exporter/awsemfexporter/metadata.yaml

```diff
@@ -7,3 +7,10 @@ status:
   distributions: [contrib, aws, observiq]
   codeowners:
     active: [Aneurysm9, shaochengwang, mxiamxia, bryan-aguilar]
+
+tests:
+  config:
+    region: 'us-west-2'
+    resource_to_telemetry_conversion:
+      enabled: true
```

Comment:
Are these required for the tests to pass?

Reply:
I checked - yes, tests fail otherwise.

Reply:
Interesting.